packages/bpk-component-bottom-sheet/src/BpkBottomSheet.tsx

```diff
@@ -31,9 +31,8 @@ import STYLES from './BpkBottomSheet.module.scss';
 
 const getClassName = cssModules(STYLES);
 
-export type Props = {
+interface CommonProps {
```

Comment:
Extract out the non-a11y props so we can conditionally allow either label or labelledBy...